test/examples-golden/positive/circuit/Polynomial/Polynomial.circuit

```diff
@@ -4,58 +4,38 @@ ArithCircuit
             ( InputWire 0 )
         , mulR = Var
             ( InputWire 0 )
-        , mulO = IntermediateWire 2
-        }
+        , mulO = IntermediateWire 2 }
     , MulGate
         { mulL = Var
             ( IntermediateWire 2 )
         , mulR = Var
             ( InputWire 0 )
-        , mulO = IntermediateWire 3
-        }
+        , mulO = IntermediateWire 3 }
     , MulGate
         { mulL = ConstGate
             ( P
-                ( 1 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 )
-            )
+                ( 1 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 ) )
         , mulR = Add
-            ( Var
-                ( IntermediateWire 3 )
-            )
+            ( Var ( IntermediateWire 3 ) )
             ( ScalarMul
                 ( P
-                    ( 52435875175126190479447740508185965837690552500527637822603658699938581184512 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 )
-                )
-                ( Var
-                    ( InputWire 1 )
-                )
-            )
-        , mulO = IntermediateWire 4
-        }
+                    ( 52435875175126190479447740508185965837690552500527637822603658699938581184512 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 ) )
+                ( Var ( InputWire 1 ) ) )
+        , mulO = IntermediateWire 4 }
     , EqualGate
         { eqI = IntermediateWire 4
         , eqM = IntermediateWire 5
-        , eqO = IntermediateWire 6
-        }
+        , eqO = IntermediateWire 6 }
     , MulGate
         { mulL = ConstGate
             ( P
-                ( 1 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 )
-            )
+                ( 1 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 ) )
         , mulR = Add
             ( ConstGate
                 ( P
-                    ( 1 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 )
-                )
-            )
+                    ( 1 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 ) ) )
             ( ScalarMul
                 ( P
-                    ( 52435875175126190479447740508185965837690552500527637822603658699938581184512 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 )
-                )
-                ( Var
-                    ( IntermediateWire 6 )
-                )
-            )
-        , mulO = OutputWire 7
-        }
-    ]
\ No newline at end of file
+                    ( 52435875175126190479447740508185965837690552500527637822603658699938581184512 `modulo` 52435875175126190479447740508185965837690552500527637822603658699938581184513 ) )
+                ( Var ( IntermediateWire 6 ) ) )
```

Comment:
Do you think is worth keeping the formatting rules for non-sexpression files as they were?

Reply:
I don't have a strong opinion either way, I can revert them if you wish. I mostly did this to avoid writing a few extra functions, as it took me a while to figure out why my tests were failing.

Reply:
We can open a separate PR if we want to change deliberately the formatting of these files

Reply:
See #855